modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm

```diff
@@ -11,3 +11,47 @@
 	armor = list(MELEE = 50, BULLET = 40, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 0, FIRE = 100, ACID = 90, WOUND = 10)
 	dog_fashion = null
 	resistance_flags = FIRE_PROOF
+
+//DS2 items
+
+/obj/item/clothing/head/helmet/swat/ds
+	name = "interdyne SWAT helmet"
+	desc = "A robust and spaceworthy helmet with a small cross on it with an 'IP' written across the earpad."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "swat_ds"
+
+/obj/item/clothing/mask/gas/syndicate/ds
+	name = "interdyne balaclava"
+	desc = "A fancy balaclava, while it doesn't muffle your voice, it's fireproof and has a miniature rebreather for internals. Comfy to boot!"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/masks.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/mask.dmi'
+	icon_state = "balaclava_ds"
+	flags_inv = HIDEHAIR | HIDEFACIALHAIR
+
+/obj/item/clothing/shoes/combat
```

Comment:
this is overwriting the original `/obj/item/clothing/shoes/combat`, comment as such above it please